conda_build/skeletons/cran.py

```diff
@@ -914,7 +914,7 @@ def skeletonize(in_packages, output_dir=".", output_suffix="", add_maintainer=No
                     print('ERROR: Version %s of package %s is archived, but --no-archive was selected' % (version, package))
                     sys.exit(1)
             elif not cran_version:
-                sys.exit("Package %s is archived; to build, an explicit version must be specified")
+                sys.exit("Package %s is archived; to build, an explicit version must be specified" % package)
```

Comment:
I think it should be ``pkg_name``  instead of ``package``, right?
```suggestion
                sys.exit("Package %s is archived; to build, an explicit version must be specified" % pkg_name)
```